### PR TITLE
2019.2 entities upgrade

### PR DESCRIPTION
**Pull request by zeroZshadow**
#### Description
Upograde to 2019.2.0f1

#### Tests
test-project upgraded

#### Documentation
[ ] Changelog

### CODE REVIEW

File: workers/unity/Packages/io.improbable.gdk.core/Systems/ComponentSendSystem.cs [hunk 1]
```diff
@@ -65,10 +65,9 @@ protected override void OnUpdate()
 
             Profiler.EndSample();
 
-            JobHandle.CompleteAll(gatheringJobs);
-
             for (var i = 0; i < componentReplicators.Count; i++)
             {
+                gatheringJobs[i].Complete();
```
Comment (paulbalaji) [inline workers/unity/Packages/io.improbable.gdk.core/Systems/ComponentSendSystem.cs]: wasn't this in the other PR

Comment (zeroZshadow) [reply workers/unity/Packages/io.improbable.gdk.core/Systems/ComponentSendSystem.cs]: Yeah it is, i'll fix this up once I rebase on develop

**Comment by zeroZshadow** [comment]
This relies on https://github.com/spatialos/gdk-for-unity-shared-ci/pull/56 being merged first, and then re-pinned.

File: ci/bootstrap.sh [hunk 2]
```diff
@@ -25,7 +25,7 @@ mkdir "${SHARED_CI_DIR}"
 pushd "${SHARED_CI_DIR}"
     git init
     git remote add origin "${CLONE_URL}"
-    git fetch --depth 20 origin master
+    git fetch --depth 20 origin feature/unity-2019.2.0f1
```
Comment (jamiebrynes7) [inline ci/bootstrap.sh]: Remember to change this back before merging 😉

Comment (zeroZshadow) [reply ci/bootstrap.sh]: This is now changed back, and uses the new bootstrap method that allows the pinned branch to be in the file

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 ### Changed
```
Comment (jamiebrynes7) [inline CHANGELOG.md]: Nit: can we put this heading below the `Added` one?

**Pull request merged**